regression-test/suites/mtmv_p0/test_iceberg_mtmv.groovy

```diff
@@ -65,5 +65,128 @@ suite("test_iceberg_mtmv", "p0,external,iceberg,external_docker,external_docker_
+        String hdfs_port = context.config.otherConfigs.get(hivePrefix + "HdfsPort")
+        String default_fs = "hdfs://${externalEnvIp}:${hdfs_port}"
+        String warehouse = "${default_fs}/warehouse"
+
+        String catalog_name = "iceberg_mtmv_catalog_hms";
+        String mvName1 = "test_iceberg_mtmv_ts"
+        String mvName2 = "test_iceberg_mtmv_d"
+        String dbName = "regression_test_mtmv_partition_p0"
+        String icebergDb = "iceberg_mtmv_partition"
+        String icebergTable1 = "tstable"
+        String icebergTable2 = "dtable"
+        sql """drop catalog if exists ${catalog_name} """
+        sql """create catalog if not exists ${catalog_name} properties (
+            'type'='iceberg',
+            'iceberg.catalog.type'='hms',
+            'hive.metastore.uris' = 'thrift://${externalEnvIp}:${hms_port}',
+            'fs.defaultFS' = '${default_fs}',
+            'warehouse' = '${warehouse}',
+            'use_meta_cache' = 'true'
+        );"""
+
+        sql """switch internal"""
+        sql """drop database if exists ${dbName}"""
+        sql """create database if not exists ${dbName}"""
+        sql """use ${dbName}"""
+
+        sql """drop table if exists ${catalog_name}.${icebergDb}.${icebergTable1}"""
+        sql """drop table if exists ${catalog_name}.${icebergDb}.${icebergTable2}"""
+        sql """create database if not exists ${catalog_name}.${icebergDb}"""
+        sql """
+            CREATE TABLE ${catalog_name}.${icebergDb}.${icebergTable1} (
+              ts DATETIME,
+              value INT)
+            ENGINE=iceberg
+            PARTITION BY LIST (DAY(ts)) ();
+        """
+        sql """insert into ${catalog_name}.${icebergDb}.${icebergTable1} values ('2024-10-26 01:02:03', 1), ('2024-10-27 01:02:03', 2), ('2024-10-27 21:02:03', 3)"""
+        sql """CREATE MATERIALIZED VIEW ${mvName1} BUILD DEFERRED REFRESH AUTO ON MANUAL partition by(`ts`) DISTRIBUTED BY RANDOM BUCKETS 2  PROPERTIES ('replication_num' = '1') as SELECT * FROM ${catalog_name}.${icebergDb}.${icebergTable1}"""
+        sql """REFRESH MATERIALIZED VIEW ${mvName1} complete"""
+        waitingMTMVTaskFinishedByMvName(mvName1)
+        qt_test_ts_refresh1 "select * from ${mvName1} order by value"
+
+        sql """insert into ${catalog_name}.${icebergDb}.${icebergTable1} values ('2024-10-26 21:02:03', 4)"""
+        sql """REFRESH MATERIALIZED VIEW ${mvName1} complete"""
+        waitingMTMVTaskFinishedByMvName(mvName1)
+        qt_test_ts_refresh2 """select * from ${mvName1} order by value"""
+
+        sql """insert into ${catalog_name}.${icebergDb}.${icebergTable1} values ('2024-10-26 01:22:03', 5), ('2024-10-27 01:12:03', 6);"""
+        sql """REFRESH MATERIALIZED VIEW ${mvName1} partitions(p_20241026000000_20241027000000);"""
+        waitingMTMVTaskFinishedByMvName(mvName1)
+        qt_test_ts_refresh3 """select * from ${mvName1} order by value"""
+
+        sql """REFRESH MATERIALIZED VIEW ${mvName1} complete"""
```

Comment:
It is necessary to test the specified auto refresh, otherwise snapshot information will not be calculated.